Rover/sailboat.cpp

```diff
@@ -102,6 +102,13 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("LOIT_RADIUS", 9, Sailboat, loit_radius, 5),
 
+    // @Param: CTRL_TYPE
+    // @DisplayName: Sail control type
+    // @Description: Select the strategy of sail control for the sailboat
+    // @Values: 0:Default SAIL_ANGLE_IDEAL from apparent wind,1:Fixed to SAIL_ANGLE_MIN
+    // @User: Standard
+    AP_GROUPINFO("CTRL_TYPE", 10, Sailboat, sail_control_type, 0),
```

Comment:
I wonder if we could think of a param name that better specifies that we are talking about the mainsail control (e..g not the rudder or lean control).

Reply:
yeah, I also thought that might be a problem. currently, the full param name is `SAIL_CTRL_TYPE`. initially, I thought of using `SAIL_CTRL`. the full name `SAIL_SAIL_CTRL` look a bit redundant to me so I changed it to `CTRL_TYPE`